base/src/main/scala/co/uproot/abandon/Ast.scala

```diff
@@ -369,4 +375,12 @@ case class Scope(entries: Seq[ASTEntry], parentOpt: Option[Scope]) extends ASTEn
         includedScopes.foreach(_.checkDupes())
     }
   }
+
+  def checkUnusedSymbols() {
+    import Console.{YELLOW, BOLD, RESET}
+    definitions.filterNot(_.isUsed)
+      .foreach(d => {
+        println(s"${YELLOW}${BOLD}symbol '${d.name}' defined in ${d.pos.filename} line: ${d.pos.pos.line} but never used${RESET}")
+      })
+  }
```

Comment:
OK, the problem is that those nested scopes we're looking for are not within the includedScopes, but childScopes. The method checkUnusedSymbols() must be called on the childScopes, too. I'm going to push my change in a moment.